core/src/main/java/org/apache/gravitino/storage/relational/service/MetalakeMetaService.java

```diff
@@ -231,7 +231,7 @@ public boolean deleteMetalake(NameIdentifier ident, boolean cascade) {
             () ->
                 SessionUtils.doWithoutCommit(
                     SecurableObjectMapper.class,
-                    mapper -> mapper.softDeleteRoleMetasByMetalakeId(metalakeId)),
+                    mapper -> mapper.softDeleteSecurableObjectsByMetalakeId(metalakeId)),
```

Comment:
Is this just a fix to a problem that existed before? I reviewed the code, L230 has the same code if it's not modified.

Reply:
One is deleting role meta data, another is deleting role's securable object metadata.